documentation/src/main/asciidoc/topics/ref_eviction_examples.adoc

```diff
@@ -35,13 +35,12 @@ create a new entry:
 
 [source,xml,options="nowrap",subs=attributes+]
 ----
-<encoding media-type="application/x-protostream"/> <1>
+<!-- Specifies a binary format for entries in the cache. -->
```

Comment:
poor example, it doesn't have its own xml file :rofl:

Reply:
yeah, good call @pruivo should be fixed now. I used `<distributed-cache ...>` as well so the config examples are more "in context"